demo/demo.md

```diff
@@ -22,39 +22,39 @@ In these examples the structure of the HTML uses the slot element to pull the tr
 
 <div class="exampleWrapper">
   <auro-popover for="button1">
-    Top popover content!
+    <fs-content-non-combinability slot="tooltip"></fs-content-non-combinability>
```

Comment:
```
<link rel="stylesheet" href="https://p2pcontent-fd-prod.azurefd.net/components/shared/component-base.css">
<script src="https://p2pcontent-fd-prod.azurefd.net/components/shared/component-base.js" type="text/javascript"></script>
<script src="https://p2pcontent-fd-prod.azurefd.net/components/fs-content-non-combinability/main.js" type="text/javascript"></script>
```

Reply:
@blackfalcon it seems like adding the above lines does not actually cause the browser to render them in the new markdown viewer. Is this a known limitation? Do you know a workaround?

Reply:
The additional script tags need to be added to the index.html file.